docs/introduction/getting-started.mdx

```diff
@@ -28,20 +32,220 @@ To get started:
 If you're on Mac, it will look something like this:
 
 ```shell
-tar zxvf conduit_0.8.0_Darwin_x86_64.tar.gz
+tar zxvf conduit_0.8.0_Darwin_arm64.tar.gz
```

Comment:
Should be 0.9 now.

Reply:
was updated in main and came into branch via merge.